JS/Lesson26/saffet/script.js

```diff
@@ -5,10 +5,99 @@ h2 (.post-title)
 p (.post-body)
 */
 
+const postArea = document.getElementById("postArea");
+const container = document.getElementById("container");
+const createButton = document.getElementById("create-btn");
+
 const URL = 'https://jsonplaceholder.typicode.com/posts';
+
+let lastPostId = 0;
+
 function fetchPosts() {
+
+    fetch(URL)
+    .then(response => response.json())
+    .then(data => {
+
+        data.forEach(post => {
+
+            lastPostId = Math.max(...data.map(post => post.id));
+            console.log("Last Post id: ", lastPostId);
+            
+            const newPost = document.createElement("div");
+            newPost.classList.add("post");
+            newPost.setAttribute("data-post-id", post.id);
+
+            const editButton = document.createElement("button");
```

Comment:
Since the button allows the user to navigate to a different page rather than perform an action on the current page, it is more semantically correct to use <a> element instead of a <button> element here